### ISSUE TITLE
[REQ]Support Shortcuts.app in Macos

### ISSUE THREAD

**Issue by yoyosbg** [aonez/Keka#1267]
### Is your feature request related to a problem? Please describe
I just want to automation keka actions in Shortcuts.app in latest Macos.

### Describe the solution you'd like
keka actions

**Comment by aonez** [comment]
You can use the `Automator.app`, does that fill your purpose?

**Comment by yoyosbg** [comment]
I give up , just use bash to do it. 

/Applications/Keka.app/Contents/MacOS/Keka --cli 7zz

**Comment by adam-jan-82** [comment]
Yes, the best solution on MacOs is running command line command either directly or via Applescript (if running in `Shortcuts.app` remember to select "Allow Running Scripts" in its preferences). 

However, on iOS there is no alternative. @aonez , would it be possible to expose basic commands like 'Archive' and 'Unarchive', specifically with password protection? I tested all zip apps in iOS AppStore (literally more than 40) and none of them implements `Shortcuts.app` zip/unzip actions that support passwords (AES encryption).
